pallets/xyk/src/lib.rs

```diff
@@ -954,6 +954,11 @@ impl<T: Config> AMM<T::AccountId, AssetId, AssetPair, Balance> for Pallet<T> {
 	fn get_max_out_ratio() -> u128 {
 		T::MaxOutRatio::get()
 	}
+
+	fn get_fee(_pool_account_id: &T::AccountId) -> (u32, u32) {
+		let fee = T::GetExchangeFee::get();
```

Comment:
so we are using `Fee::Default()` of 0,2% for any xyk pool? 
just double checking if this is the intention, bc in LBP it is customisable

Reply:
It's customisable in the config, but for all pools.